dvc/output/local.py

```diff
@@ -38,7 +39,15 @@ def _parse_path(self, remote, path):
         return self.REMOTE.path_cls(abs_p)
 
     def __str__(self):
-        return str(self.path_info)
+        if not self.is_in_repo:
+            return str(self.def_path)
+
+        root_dir = os.path.join(self.repo.root_dir, "")
+        curr_dir = os.getcwd()
+        if curr_dir.startswith(root_dir):
```

Comment:
This pattern is used over and over in our code. This is probably a good time to extract this into some `path_isin()` utility.

Reply:
Will do.